app/web/features/communities/events/EventsSection.tsx

```diff
@@ -56,7 +76,6 @@ export default function EventsSection({
           <HorizontalScroller className={classes.cardContainer}>
             {data.pages
               .flatMap((page) => page.eventsList)
-              .filter((event) => !event.isCancelled)
```

Comment:
I think we probably want to leave the functionality the same here! People can go to the events page if they want to see their cancelled events. We just won't see the canceled version in the communities page.